dd-java-agent/instrumentation/javax-mail/src/main/java/datadog/trace/instrumentation/javax/mail/JavaxMailInstrumentation.java

```diff
@@ -0,0 +1,78 @@
+package datadog.trace.instrumentation.javax.mail;
+
+import static datadog.trace.agent.tooling.bytebuddy.matcher.NameMatchers.named;
+
+import com.google.auto.service.AutoService;
+import datadog.trace.agent.tooling.Instrumenter;
+import datadog.trace.agent.tooling.InstrumenterModule;
+import datadog.trace.api.iast.InstrumentationBridge;
+import datadog.trace.api.iast.Sink;
+import datadog.trace.api.iast.VulnerabilityTypes;
+import datadog.trace.api.iast.sink.EmailInjectionModule;
+import java.io.IOException;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Part;
+import javax.mail.internet.MimeMultipart;
+import net.bytebuddy.asm.Advice;
+
+@AutoService(InstrumenterModule.class)
+public class JavaxMailInstrumentation extends InstrumenterModule.Iast
+    implements Instrumenter.ForSingleType, Instrumenter.HasMethodAdvice {
+
+  public JavaxMailInstrumentation(String instrumentationName, String... additionalNames) {
+    super("javaxmailinstrumentation", "javaxmail");
```

Comment:
```suggestion
    super("javax-mail", "transport");
```
For most of our instrumentations the first argument is the library name and the additional name it's related with the instrumented class